composer.json

```diff
@@ -32,7 +32,8 @@
     "sendgrid/sendgrid": "~7",
     "slim/php-view": "^2.2",
     "slim/psr7": "^1.1.0",
-    "slim/slim": "^4.7"
+    "slim/slim": "^4.7",
+    "twbs/bootstrap-icons": "^1.11"
```

Comment:
this is in the package.json - does it need to be here too? this will install it twice surely?